tools/rapids-upload-docs

```diff
@@ -71,16 +71,30 @@ copy_docs_to_s3() {
     for PROJECT_FORMAT_DIR in "${PROJECT_DIR}"/*; do
       FORMAT=$(basename "${PROJECT_FORMAT_DIR}")
 
-      if [[ ! "${FORMAT}" =~ ^(html|txt)$ ]]; then
-        echo "ERROR: FORMAT must be either 'html' or 'txt'."
+      if [[ ! "${FORMAT}" =~ ^(html|txt|xml_tar)$ ]]; then
+        echo "ERROR: FORMAT must be either 'html' or 'txt' or 'xml_tar'."
         exit 1
       fi
 
+      if [[ "${FORMAT}" == "xml_tar" ]]; then
+        NUM_FILES=$(find "$PROJECT_FORMAT_DIR" -maxdepth 1 -type f | wc -l)
```

Comment:
```suggestion
        NUM_FILES=$(find "$PROJECT_FORMAT_DIR" -type f | wc -l)
```

Do we need `-maxdepth`? seems safer to omit it in case there are subdirectories (which we don't want)